client/src/main/java/org/apache/rocketmq/client/trace/AsyncTraceDispatcher.java

```diff
@@ -73,14 +73,16 @@
     private String traceTopicName;
     private AtomicBoolean isStarted = new AtomicBoolean(false);
     private AccessChannel accessChannel = AccessChannel.LOCAL;
+    private String producerOrConsumerGroup;
 
-    public AsyncTraceDispatcher(String traceTopicName, RPCHook rpcHook) {
+    public AsyncTraceDispatcher(String producerOrConsumerGroup, String traceTopicName, RPCHook rpcHook) {
```

Comment:
I think it will be better if the variable "producerOrConsumerGroup" change to be "group" name.